src/libcadet/HighResKoren.hpp

```diff
@@ -0,0 +1,217 @@
+// =============================================================================
```

Comment:
probably we should clean up the commit history a bit and make this a separate commit.